srcpkgs/apparmor/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'apparmor'
 pkgname=apparmor
 version=2.13.2
-revision=2
+revision=3
 wrksrc="${pkgname}-v${version}"
 build_wrksrc=libraries/libapparmor
 build_style=gnu-configure
-conf_files="/etc/apparmor.d/local/*"
+conf_files="/etc/apparmor.d/local/* /etc/apparmor/*"
```

Comment:
Why modify stuff in /etc/apparmor when you have /etc/apparmor.d/local?

Reply:
/etc/apparmor.d/local only contains apparmor profiles, not configuration of apparmor itself, which goes to /etc/apparmor/. For example, if I want to speed up boot on system with apparmor I need to put `write-cache` into /etc/apparmor/parser.conf. Same with any other apparmor specific configuration.

Reply:
Ah, alright.